README.md

```diff
@@ -163,3 +163,12 @@ This comparison describes only free tiers of these services.
 ]
 ```
 </details>
+
+## Zeabur.com
+
+[Pricing](https://zeabur.com/pricing) | [Deployment](https://docs.zeabur.com/get-started)
```

Comment:
I think it would be nice to add a link to the Pricing page in the docs

```suggestion
[Pricing](https://zeabur.com/pricing) | [Pricing Model](https://docs.zeabur.com/billing/pricing) | [Deployment](https://docs.zeabur.com/get-started)